src/Microsoft.Sbom.Api/Utils/Constants.cs

```diff
@@ -28,6 +33,22 @@ public static class Constants
         Version = "1.0.0"
     };
 
+    public static SbomSpecification SPDX22Specification = ManifestInfo.ToSBOMSpecification(SPDX22ManifestInfo);
+
+    public static Collection<SbomSpecification> SupportedSbomSpecifications = new()
+    {
+        ManifestInfo.ToSBOMSpecification(SPDX22ManifestInfo),
+        ManifestInfo.ToSBOMSpecification(SPDX30ManifestInfo),
+        ManifestInfo.ToSBOMSpecification(TestManifestInfo),
```

Comment:
I'm guessing that this is here so that the code in `BaseManifestInfo.ConfigHandler` can use our test cases? This is fine for now, but we should change this before releasing. Simplest option may be to have the tests pass in a custom `IConfiguration` when building the `*ConfigHandler`, so that we allow `TestManifestInfo` in tests but not in shipping code

Reply:
This actually does not affect the unit tests. I added it as a precaution for future unit tests but didn't realize it couldn't be shipped.